README.md

```diff
@@ -42,7 +42,7 @@ This is the other way to get the bot into one of your servers, and is best for d
 - Clone the repo
 - Copy `config.example.json` to `config.json`
 - In `config.json`, provide values for at least `botToken`, `homeChannel` and `homeServerId`. Other values are optional, but some functionality may not work without them (the bot should tell you what's missing though if you try to use a feature that requires a value you didn't set). If you're not sure about a value, see [the wiki page on config.json](https://github.com/FloatingMilkshake/MechanicalMilkshake/wiki/Configuration#configjson) or feel free to contact me!
-- Run `docker-compose up -d`
+- Run `docker-compose up -d` 
```

Comment:
huh?

Reply:
as the commit says, dwbi.

Reply:
I'm not much of a fan of pointless commits... what was this for?

Reply:
It was to see if I could do a squash to my repo that would remove all of #4's commits from this PR but it didn't work lmao

Reply:
fair enough